package.json

```diff
@@ -19,25 +19,15 @@
       "require": "./lib/index.js",
       "import": "./lib/index.js"
     },
-    "./future": {
-      "types": "./future.d.ts",
-      "require": "./future.js",
-      "import": "./future.js"
-    },
     "./types": {
-      "types": "./types.d.ts",
-      "require": "./types.js",
-      "import": "./types.js"
+      "types": "./typings/telegram-types.d.ts"
```

Comment:
The empty JS files are there so ESLint doesn't complain about importing a missing file when there's only a `d.ts`.

Ref: https://t.me/TelegrafJSChat/67718

Reply:
This is no longer necessary. Telegraf 5 requires Node 18, which supports `package.json` `"exports"`.